docs/reference/config.rst

```diff
@@ -716,6 +716,17 @@ Default: ``{}`` (empty).
 MusicBrainz Options
 -------------------
 
+.. _musicbrainz.enabled:
+
+enabled
+~~~~~~~
+
+This option allows you to disable using MusicBrainz as a metadata source. This applies
+if you use plugins that fetch data from alternative sources and should make the import
+process quicker.
+
+Default: ``yes``.
```

Comment:
I think this documentation stuff might be out of order—it makes the "top-level" description below appear to be part of the "enabled" subsection. I think this should appear among the other subsections below (which start with "searchlimit").

Reply:
Agree - have moved it below.